products/sentinel_one.py

```diff
@@ -71,17 +124,18 @@ def __init__(self, profile: str, creds_file: str, account_id: Optional[list[str]
         account_names = account_name if account_name else list()
 
         # extract account/site ID from configuration if set
-        if 'account_id' in config[profile] and config[profile]['account_id'] not in account_ids:
-            account_ids.append(config[profile]['account_id'])
+        if 'account_id' in config[self.profile] and config[self.profile]['account_id'] not in account_ids:
+            account_ids.append(config[self.profile]['account_id'])
 
-        if 'site_id' in config[profile] and config[profile]['site_id'] not in site_ids:
-            site_ids.append(config[profile]['site_id'])
+        if 'site_id' in config[self.profile] and config[self.profile]['site_id'] not in site_ids:
+            site_ids.append(config[self.profile]['site_id'])
 
-        if 'account_name' in config[profile] and config[profile]['account_name'] not in account_names:
-            account_names.append(config[profile]['account_name'])
+        if 'account_name' in config[self.profile] and config[self.profile]['account_name'] not in account_names:
+            account_names.append(config[self.profile]['account_name'])
```

Comment:
```suggestion
        if 'account_name' in config[self.profile]:
            for name in config[self.profile]['account_name'].split(','):
                if name not in account_names:
                    account_names.append(name)
```
add a check similar to this in order to allow multiple account_names to be specified in a config file.